book/05-distributed-git/sections/contributing.asc

```diff
@@ -90,14 +90,6 @@ Further paragraphs come after blank lines.
 If all your commit messages follow this model, things will be much easier for you and the developers with whom you collaborate.
 The Git project has well-formatted commit messages -- try running `git log --no-merges` there to see what a nicely-formatted project-commit history looks like.
 
-[NOTE]
-.Do as we say, not as we do.
-====
-For the sake of brevity, many of the examples in this book don't have nicely-formatted commit messages like this; instead, we simply use the `-m` option to `git commit`.
-
-In short, do as we say, not as we do.
-====
-
```

Comment:
Let's actually keep this, since we're still using the `-m` flag all over the place.

Reply:
Hmm that took a bit of work to figure out how to get that bit out of the commit... 😄 

Managed to do it with:
1. `git rebase -i upstream/master`, selecting `edit` on the commit.
2. First stage all the known good stuff with git add.
3. `git add -p book/05-distributed-git/sections/contributing.asc` to stage the good stuff from that section.
4. `git restore book/05-distributed-git/sections/contributing.asc` to leave the single bad part behind.